portal/templates/initial_queries_macros.html

```diff
@@ -131,9 +131,9 @@ <h4 class="modal-title" id="termsReminderModalLabel">{{ _("To Continue") }}</h4>
     {{tou_agreement_url()}}
     <script>
       $(function() {
-          tnthAjax.getTerms({{user.id}}, false, false, function() {
+          tnthAjax.getTerms($("#iq_userId").val(), false, false, function() {
```

Comment:
Would you be able to move this into `initialQueries.js` and check for `$('#iq_userId')` from there?